src/components/BulkOperations/BulkOperations.scss

```diff
@@ -25,3 +25,9 @@
         }
     }
 }
+
+#bulk-operation-historical-records {
+    .pgn__smart-status {
+        display: none;
+    }
+}
```

Comment:
[curious] Why is this required?

Reply:
The table display the `Clear Filters` option. However, on clearing the filters, the dropdown is not updated. That seems like Bad UX, and we don't really need that option, hence I hid it :-)